### PR TITLE
Tweaks

### PR DESCRIPTION
**Pull request by kevmoo**
[Review on Reviewable](https://reviewable.io/reviews/sethladd/dart_to_js_script_rewriter/11)

### CODE REVIEW

**Comment by sethladd** [comment]
ah, drat. I just saw this. Sorry for missing it :(

**Comment by markrusciano-wf** [comment]
+1

**Comment by sethladd** [comment]
If someone can fix this patch I will merge it.

**Comment by kevmoo** [comment]
@sethladd merge away!

File: test/all.dart [hunk 1]
```diff
@@ -24,7 +24,7 @@ void defineTests() {
       DartToJsScriptRewriter transformer = _transformer();
       Document document = new Document.html(_html);
       transformer.rewriteDartTags(document);
-      List dartJsScripts = document.querySelectorAll('script').where((tag) {
+      var dartJsScripts = document.querySelectorAll('script').where((tag) {
```
Comment (sethladd) [inline test/all.dart]: FWIW I'm not a fan of this change. Relying on type dynamic gets you into trouble when refactoring. If you change how dartJsScripts is initialized later, the variable is still dynamic and you'll run into bugs. As systems grow, type annotations are beautiful.

Comment (kevmoo) [inline test/all.dart]: Understood. If we're running analyzer as part of the CI process, this will be caught I think.

**Comment by maxwellpeterson-wf** [comment]
+1

**Pull request merged**